src/Driver/Mysqli/Driver.php

```diff
@@ -32,27 +34,9 @@ public function connect(array $params)
             $host = $params['host'] ?? null;
         }
 
-        $flags = 0;
-
-        $preInitializers = $postInitializers = [];
-
-        if (isset($params['driverOptions'])) {
-            $driverOptions = $params['driverOptions'];
-
-            if (isset($driverOptions[Connection::OPTION_FLAGS])) {
-                $flags = $driverOptions[Connection::OPTION_FLAGS];
-                unset($driverOptions[Connection::OPTION_FLAGS]);
-            }
-
-            $preInitializers = $this->withOptions($preInitializers, $driverOptions);
-        }
-
-        $preInitializers  = $this->withSecure($preInitializers, $params);
-        $postInitializers = $this->withCharset($postInitializers, $params);
-
```

Comment:
What's the link with this refactoring and marking parameters as sensitive?

Reply:
I would need to flag the initializers arrays as sensitive because one of them wraps an SSL key. I wanted to avoid that.